docs/source/index.rst

```diff
@@ -166,6 +173,7 @@ messy situations in everyday problems.
    develop.rst
    changelog.rst
    configuration.rst
+   how-to/index.rst
```

Comment:
A "how to" cookbook type section is one of the fundamental sections in Diataxis. Right now we don't have much of this content and it's become a bit of a catchall for stuff we haven't put in the right place yet. But as we go forwards I want to make more of this section, not less of it.

I don't think demoting this is the right move.

Reply:
I think the main value of the how-to is to be able to point directly to snippets. The list of all the pages is not as interesting to me. So I'd actually be fine just removing it from the toc for the time being and trying to plump it up a little before we promote it.

Reply:
I think that if we have anything at this high level of visibility then it needs to be high quality.   This page is not currently high quality.  I think that we should remove this and then invest in it and then bring it back once that investment is done.

Reply:
It seems like @jsignell is ok with removing this until it becomes higher quality.  Leaving this in for now.